ui/src/kapacitor/components/RuleMessage.js

```diff
@@ -55,17 +55,25 @@ export const RuleMessage = React.createClass({
       return {text, ruleID: rule.id}
     }).concat(defaultAlertEndpoints)
 
-    const selectedAlert = rule.alerts[0]
+    const selectedAlert = rule.alerts[0] || alerts[0].text
 
     return (
       <div className="kapacitor-rule-section">
         <h3 className="rule-section-heading">Alert Message</h3>
         <div className="rule-section-body">
-          <div className="rule-section--item alert-message--endpoint">
-            <div>
-              <p>Send this Alert to:</p>
-              <Dropdown className="dropdown-kapacitor size-136" selected={selectedAlert || 'Choose an output'} items={alerts} onChoose={this.handleChooseAlert} />
-            </div>
+          <div className="kapacitor-values-tabs">
+            <p>Send this Alert to:</p>
+            <ul className="btn-group btn-group-lg tab-group">
+              {alerts.map(alert =>
```

Comment:
haha, you'd lose the implicit return by adding a block and a return and it's just not worth it at that point. good point!